lib/component-dependencies.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
```

Comment:
Does this get built automatically? If so, maybe we should exclude the lib directory and not check this in.